caddyhttp/httpserver/logger.go

```diff
@@ -84,6 +85,17 @@ func (l Logger) MaskIP(ip string) string {
 
 }
 
+func (l Logger) ShouldLog(path string) bool {
+	shouldLog := true
+	for ex := 0; ex < len(l.Exceptions); ex++ {
+		if Path(path).Matches(l.Exceptions[ex]) {
+			shouldLog = false
+			break
```

Comment:
Replace with `return false` ;) - then you can get rid of `shouldLog`

Reply:
Done.